packages/dds/tree/src/feature-libraries/modular-schema/fieldChangeHandler.ts

```diff
@@ -245,12 +243,9 @@ export type ValueChange =
 			revision?: RevisionTag;
 
 			/**
-			 * The tag of the change that overwrote the value being restored.
-			 *
-			 * Undefined when the operation is the product of a tag-less change being inverted.
-			 * It is invalid to try convert such an operation to a delta.
+			 * The value being restored.
 			 */
-			revert: RevisionTag | undefined;
+			revert: Value;
```

Comment:
This is the one case where I think we don't need to differentiate between the "set" and "revert" case, so I think we can just collapse this union type into `{ revision?: RevisionTag; value?: Value; }`.
Sorry I didn't think of this when we talked about revive vs. insert stuff.